src/pages/Account/components/AccountDetailComponent.tsx

```diff
@@ -22,98 +21,100 @@ const AccountDetailComponent: FC<AccountProps> = ({ accountId, className }) => {
 
   if (isAccountFetching) return <LoadingComponent />;
 
-  const { available_balance: availableBalance,
+  const {
+    available_balance: availableBalance,
     free_balance: freeBalance,
     locked_balance: lockedBalance,
-    timestamp } = account || {};
+    timestamp
+  } = account || {};
 
   return (
-    <div className={className}>
-      <div className={'account-container'}>
-        <div className={'avatar-container'}>
-          <Avatar
-            size='large'
-            value={accountId}
-          />
-        </div>
-        <div className={'name-container'}>
-          <Text size={'l'}>Account name</Text>
-          <h2>
-            {deviceSize === DeviceSize.sm || deviceSize === DeviceSize.md
-              ? shortcutText(accountId)
-              : accountId}
-          </h2>
-        </div>
-        <Text
-          color={'grey-500'}
-        >
+    <AccountContainer>
+      <div>
+        <Avatar
+          size='large'
+          value={accountId}
+        />
+      </div>
+      <div>
+        <Text size={'l'}>Account name</Text>
+        <h2>
+          {deviceSize === DeviceSize.sm || deviceSize === DeviceSize.md
+            ? shortcutText(accountId)
+            : accountId}
+        </h2>
+      </div>
+      <Text
+        color={'grey-500'}
+      >
           Created on
-        </Text>
-        <Text>
-          {timestamp ? new Date(timestamp).toLocaleString() : 'unavailable'}
-        </Text>
-        <Text color={'grey-500'}>
+      </Text>
+      <Text>
+        {timestamp ? new Date(timestamp).toLocaleString() : 'unavailable'}
+      </Text>
+      <Text color={'grey-500'}>
           Balance
-        </Text>
-        <div className={'balance-container'}>
-          <Text>{`${freeBalance || 'unavailable'} ${
-            chainData?.properties.tokenSymbol || ''
-          } (total) `}</Text>
-          <Text color={'grey-500'}>{`${lockedBalance || 'unavailable'} ${
-            chainData?.properties.tokenSymbol || ''
-          } (locked) `}</Text>
-          <Text color={'grey-500'}>{`${availableBalance || 'unavailable'} ${
-            chainData?.properties.tokenSymbol || ''
-          } (transferable)`}</Text>
-        </div>
-      </div>
-    </div>
+      </Text>
+      <BalanceContainer>
+        <Text>{`${freeBalance || 'unavailable'} ${
+          chainData?.properties.tokenSymbol || ''
+        } (total) `}</Text>
+        <Text color={'grey-500'}>{`${lockedBalance || 'unavailable'} ${
+          chainData?.properties.tokenSymbol || ''
+        } (locked) `}</Text>
+        <Text color={'grey-500'}>{`${availableBalance || 'unavailable'} ${
+          chainData?.properties.tokenSymbol || ''
```

Comment:
it's always better to extract props before moving into jsx, so you end up with "tokenSymbol" where at the beginning of component, smth like this:
const chainData
const properties = chainData?.properties || {}
const { tokenSymbol } = properties;